test/CodeGen/AVR/ctlz.ll

```diff
@@ -0,0 +1,10 @@
+; RUN: llc < %s -march=avr | FileCheck %s
+
+define i8 @count_leading_zeros(i8) unnamed_addr {
+; CHECK-LABEL: count_leading_zeros:
```

Comment:
I would be inclined to pretty much copy that entire output (if it's known to be correct), replace registers with regexes, and bam.

Other backends do things this way too. It's uncommon for codegen to change for things like this - this entire sequence of instructions is probably two or so DAG nodes - not much wiggleroom.

Aside from that, if something does change, it'll likely be subtle and easy to fix the test up.